build/common_functions

```diff
@@ -184,3 +184,34 @@ function GetDistro {
     fi
     export DISTRO
 }
+
+function CompareVersionStrings () {
```

Comment:
This code is copypasted from http://stackoverflow.com/questions/4023830/bash-how-compare-two-strings-in-version-format and *cannot* be used in this project because `cc by-sa 3.0` is not compatible with GNU GPLv3. Furthermore, this function is useless for this change.